### PR TITLE
fix for #1629

### PR DESCRIPTION
**Pull request by kandizzy**
Modified marker-child-model.coffee to confirm `click` is a function.

### CODE REVIEW

**Comment by nmccready** [comment]
Are the number of file changes necessary? Looks like you PR'd from a old old branch. Changelog is out of date.

File: src/coffee/directives/api/models/child/marker-child-model.coffee [hunk 1]
```diff
@@ -47,7 +47,6 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         else
           action = new PropertyAction (calledKey, newVal) =>
```
Comment (nmccready) [inline src/coffee/directives/api/models/child/marker-child-model.coffee]: Is this the actual change? Is so just PR marker-child-model.coffee. I will rebuild it post merge.

**Comment by kandizzy** [comment]
just this line
https://github.com/kandizzy/angular-google-maps/blob/master/src/coffee/directives/api/models/child/marker-child-model.coffee#L221

**Comment by nmccready** [comment]
Ok if you can re-PR it with just that change it would be appreciated.

**Comment by kandizzy** [comment]
ok

**Pull request closed without merging**